scylladb/care-pet
# docs: moved latest-version to conf.py

**Related issues:** https://github.com/scylladb/sphinx-scylladb-theme/issues/87

All the variables related to the multiversion extension will be defined in ``conf.py`` instead of using the GitHub Actions file.

##  How to test this PR

Run ``make multiversionpreview``. When you open https://0.0.0.0:5500,  the browser should render the documentation.
